src/test/e2e/tests/specs.test.ts

```diff
@@ -275,62 +228,64 @@ if (process.env.EXCLUDE_E2E === 'true') {
       expect(await mediaAccountRequestSubmittedPage.getPanelTitle()).toEqual('Details submitted');
     });
   });
-}
-if (process.env.EXCLUDE_E2E === 'true') {
-  describe('Verified user', () => {
-    describe('Sign In Page', () => {
-      const HMCTSAccountUrl = 'https://hmcts-sjp.herokuapp.com/sign-in-idam.html';
+});
+
+describe('Verified user', () => {
+  describe('Sign In Page', () => {
+    const HMCTSAccountUrl = 'https://hmcts-sjp.herokuapp.com/sign-in-idam.html';
```

Comment:
is this still needed? is this not from stitching?

Reply:
After talking with Alberto we decided to remove 2 disabled steps